spritz/spritz-android/src/main/java/com/novoda/spritz/SpritzPage.java

```diff
@@ -0,0 +1,79 @@
+package com.novoda.spritz;
+
+import android.animation.TimeInterpolator;
+
+import java.util.concurrent.TimeUnit;
+
+public class SpritzPage {
```

Comment:
This reads "page" but it just contains only info about animation :/

Reply:
Maybe `Step` is clearer and clashes less with the `ViewPager`'s `Page` domain?

Reply:
Done in e2a51cdb19e2d3e97f72ab0f246ac572131838bd! 💯